controller/comments.controller.js

```diff
@@ -25,5 +25,12 @@ const postComment = (req, res, next) => {
     ).catch((error) => next(error));
 };
 
+const deleteComments = (req, res, next) => {
+    const comment_id = req.params.comment_id;
+    removeComment(comment_id).then(() => {
+        res.status(204).send({})
```

Comment:
When using a 204 status nothing is sent back no need to send or test for anything being sent back - this is convention for a delete endpoint